spec/chipmunk/status_cli_spec.rb

```diff
@@ -3,6 +3,19 @@
 require "spec_helper"
 require "chipmunk/status_cli"
 
+class ResponseDouble
```

Comment:
In case you're wondering why I did this instead of using an rspec double, it's because I kept getting `OutsideOfExampleError` exceptions from rspec for reasons that were unclear to the point that I was looking through rspec source code. I caught myself doing that, decided this was just not important enough to warrant that, and wrote this class instead.